bundles/org.openhab.binding.hapero/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,546 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hapero"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="haperoBridge">
```

Comment:
I am not sure if i understand what you mean. From what i understood so far, a bridge does not represent anything, its just how openhab calls the software part which transmits data to and from a physical device ?

Reply:
Do you mean "Watcher" because it watches the data file that the furnace creates for changes ? 
OK, but if i where a end user who does not know this detail that name would probably confuse me (Its the Bridge, why is it called "Watcher" ?) But i can change it if you like, altough i would prefer sticking with the current name for the above reason.